react/features/remote-video-menu/components/MuteButton.js

```diff
@@ -60,9 +81,26 @@ class MuteButton extends Component {
                 displayClass = { muteConfig.muteClassName }
                 iconClass = 'icon-mic-disabled'
                 id = { `mutelink_${participantID}` }
-                onClick = { onClick } />
+                onClick = { this._onClick } />
         );
     }
+
+    /**
+     * Dispatches a request to mute the participant with the passed in
+     * participantID.
+     *
+     * @private
+     * @returns {void}
+     */
+    _onClick() {
+        const { dispatch, onClick, participantID } = this.props;
+
+        dispatch(muteRemoteParticipant(participantID));
+
+        if (onClick) {
+            onClick();
+        }
+    }
```

Comment:
MuteButton and KickButton hookup into redux is not necessary but was done to help clean up what needs to get passed into RemoteVideoMenuTriggerButton/RemoteVideoMenu. In general the remote menu buttons should be smart buttons that know how to dispatch actions that affect a participant.